### PR TITLE
[ID] Update "How you can help!"

### PR DESCRIPTION
**Pull request by mousesalt**

### CODE REVIEW

File: wiki/How_you_can_help!/id.md [hunk 1]
```diff
@@ -1,84 +1,67 @@
----
-outdated: true
----
+# Bagaimana Kamu dapat membantu!
 
-# Bagaimana Anda dapat membantu!
+osu! memiliki komunitas yang sangat dinamis yang terdiri dari orang-orang dari seluruh dunia. Dan siapa pun dapat menjadi bagian dari komunitas dengan caranya sendiri!
```
Comment (wowcake) [inline wiki/How_you_can_help!/id.md:3]:
```suggestion
osu! memiliki komunitas yang sangat dinamis yang terdiri dari orang-orang dari seluruh dunia. Dan siapa pun dapat menjadi bagian dari komunitas itu dengan caranya sendiri!
```

File: wiki/How_you_can_help!/id.md [hunk 2]
```diff
@@ -1,84 +1,67 @@
...
 
-*Sebelumnya ini adalah kiriman yang [peppy](https://osu.ppy.sh/users/2) buat beberapa waktu lalu dan diminta untuk menjadikannya sebuah artikel di sini. Anda dapat melihat kiriman aslinya [di sini](https://osu.ppy.sh/community/forums/topics/83155).*
+Di bawah ini hanyalah beberapa cara yang dapat Kamu lakukan untuk membantu dalam komunitas:
 
-Saya sudah lama ingin memposting hal ini untuk sementara, tetapi selalu tertunda di saat Saya menyiapkan segala sesuatunya di situs web dan di kepala saya. Seperti yang mungkin Anda sadari, komunitas osu! beserta ekosistemnya telah tumbuh stabil, tetapi tim pengembang belum tumbuh dalam ukuran sama sekali. Saya yakin dengan komunitas sebesar ini, pasti banyak bakat tersembunyi di luar sana, jadi tujuan kiriman ini adalah untuk menjangkau dan memberi tahu Anda semua bahwa kami selalu mencari bantuan dan masukan, serta menguraikan situasi saat ini di mana Anda dapat langsung terjun dan membantu!
+## Membantu wiki
```
Comment (wowcake) [inline wiki/How_you_can_help!/id.md:7]:
```suggestion
## Membantu dalam wiki
```

File: wiki/How_you_can_help!/id.md [hunk 3]
```diff
@@ -1,84 +1,67 @@
...
 
-Pertama, dari perspektif pribadi saya sudah benar-benar jenuh. Saya menghabiskan semua waktu luang saya untuk osu! (atau osu!stream) dan tidak memiliki sumber daya lain lagi yang dapat saya andalkan. Ini berarti bahwa sebagian besar hal-hal yang tersedia di osu! saat ini didasarkan pada prioritas pribadi saya. Ini bukan berarti bahwa itu harus menjadi faktor pembatas - ada begitu banyak peluang untuk membuat osu! lebih baik tanpa perlu akses atau kemampuan seorang pengembang.
+[Wiki](https://github.com/ppy/osu-wiki) adalah basis pengetahuan resmi kami yang berisi apa saja dan segalanya tentang osu!. Oleh karena itu, wiki selalu mencari lebih banyak kontributor untuk membantu menjaga situs tetap mutakhir dengan pengetahuan maupun terjemahan yang segar dan baru.
```
Comment (wowcake) [inline wiki/How_you_can_help!/id.md:9]:
```suggestion
[Wiki](https://github.com/ppy/osu-wiki) adalah basis pengetahuan resmi kami yang berisi apa saja dan segalanya tentang osu!. Oleh karena itu, wiki selalu membutuhkan lebih banyak kontributor untuk membantu menjaga situs tetap mutakhir dengan pengetahuan maupun terjemahan yang baru dan segar.
```

File: wiki/How_you_can_help!/id.md [hunk 4]
```diff
@@ -1,84 +1,67 @@
...
-- Pantau terus [Pull Requests](https://github.com/ppy/osu-wiki/pulls) sesering mungkin (markahi :p), dan pastikan orang tidak menambahkan informasi yang tidak berguna atau salah dengan meninjau. Pantau terus apa yang sedang dikerjakan dan ikut berkontribusi jika Anda bisa!
-- Pastikan informasi di wiki itu singkat, jelas, dan seindah mungkin. Rapikan semua masalah format yang Anda temukan, dan dorong orang lain ke arah yang benar jika diperlukan.
-- Buat kiriman ini menjadi halaman wiki :D! Terjemahkan ke lebih banyak bahasa - kami membutuhkan bantuan bukan hanya dari penulis yang berbahasa Inggris (kenyatannya: lebih dari 75% komunitas osu! Tidak berbicara bahasa Inggris dan/atau mengerti bahasa Inggris sepenuhnya).
+- Ikuti kanal favorit Kamu di [obrolan osu! dalam gim](/wiki/Chat_Console), atau di forum.
+- Jika Kamu melihat sesuatu terjadi yang tidak pantas atau mengganggu, gunakan perintah `!report` dalam obrolan dalam gim untuk memberi tahu moderator.
```
Comment (wowcake) [inline wiki/How_you_can_help!/id.md:20]:
```suggestion
- Jika Kamu melihat sesuatu yang tidak pantas atau mengganggu terjadi, gunakan perintah `!report` dalam obrolan dalam gim untuk memberi tahu moderator.
```

File: wiki/How_you_can_help!/id.md [hunk 5]
```diff
@@ -1,84 +1,67 @@
...
 
-## Bantu jaga osu! tetap ramah dan aman!
+## Bantu anggota komunitas lainnya
 
-Kami memiliki moderator yang terus mencoba melakukan yang terbaik untuk menjaga obrolan tetap terkendali, tetapi terdapat 25 channel obrolan (dan terus bertambah) dan bahkan lebih banyak lagi forum, sehingga sulit bagi kami untuk mencakup semua aktivitas setiap menit dan harinya, Anda dapat membantu kami dengan sangat mudah!
+Jika Kamu menemui seseorang yang memiliki masalah mengenai gim, bantulah mereka! Apakah itu masalah pada driver, pertanyaan beatmap, atau pertanyaan gameplay, jika Kamu dapat menjawabnya, Kamu dapat membantu orang tersebut memecahkan masalah mereka.
```
Comment (wowcake) [inline wiki/How_you_can_help!/id.md:24]:
```suggestion
Jika Kamu menemui seseorang yang memiliki masalah dengan osu!, bantulah mereka! Apakah itu masalah pada driver, pertanyaan beatmap, atau pertanyaan gameplay, jika Kamu dapat menjawabnya, Kamu dapat membantu orang tersebut memecahkan masalah mereka.
```

File: wiki/How_you_can_help!/id.md [hunk 6]
```diff
@@ -1,84 +1,67 @@
...
 
-### Dimulai dari mana
+Secara garis besar, berikut ini beberapa cara Kamu dapat membantu orang lain:
```
Comment (wowcake) [inline wiki/How_you_can_help!/id.md:26]:
```suggestion
Secara garis besar, berikut ini adalah beberapa cara Kamu dapat membantu orang lain:
```

File: wiki/How_you_can_help!/id.md [hunk 7]
```diff
@@ -1,84 +1,67 @@
...
-- Ikuti channel favorit Anda di obrolan dalam game, dan forum di web.
-- Jika Anda melihat sesuatu terjadi yang mengganggu Anda (atau secara umum dianggap tidak pantas), gunakan perintah `!report` Untuk memberi tahu moderator.
-- Anda dapat menggunakan perintah `!report` untuk segala sesuatu yang terjadi di mana saja di osu!. Laporan Anda akan selalu dibaca.
+- Pilih forum yang menarik minat Kamu:
+    - [Help](https://osu.ppy.sh/community/forums/5)
+    - [Mapping Discussion](https://osu.ppy.sh/community/forums/56)
+    - [Storyboarding](https://osu.ppy.sh/community/forums/20)
+    - [Gameplay & Rankings](https://osu.ppy.sh/community/forums/13)
+- Tandai forum ini sebagai telah dibaca menggunakan tombol tautan di paling bawah.
+- Ikuti utas baru apa pun, baca masing-masing utas dan bantu bila Kamu bisa; Kamu dapat menggunakan *unread icon* untuk memastikan Kamu mengikuti.
```
Comment (wowcake) [inline wiki/How_you_can_help!/id.md:34]:
```suggestion
- Ikuti thread-thread yang terbaru, baca masing-masing thread itu dan bantu bila Kamu bisa; Kamu dapat menggunakan *unread icon* untuk memastikan Kamu mengikutinya.
```

File: wiki/How_you_can_help!/id.md [hunk 8]
```diff
@@ -1,84 +1,67 @@
...
+- Baca [panduan](/wiki/Guides) yang tersedia untuk menyempurnakan pengetahuan osu!. Jika menurutmu panduan tidak memuaskan, maka Kamu dapat mencoba berkontribusi untuk mereka!
```
Comment (wowcake) [inline wiki/How_you_can_help!/id.md:35]:
```suggestion
- Baca [panduan](/wiki/Guides) yang tersedia untuk menyempurnakan pengetahuan osu!. Jika menurutmu panduan tersebut tidak memuaskan, maka Kamu dapat mencoba berkontribusi pada panduan itu!
```

File: wiki/How_you_can_help!/id.md [hunk 9]
```diff
@@ -1,84 +1,67 @@
...
 
-## Bantu menjawab pertanyaan dukungan teknis, beatmap, dan gameplay!
+## Bantu memeriksa/modding beatmap!
 
-Setiap hari saya mencoba untuk mengurus pertanyaan-pertanyaan dukungan teknis yang luar biasa, tetapi kebanyakan pertanyaan yang saya terima dapat dijawab oleh siapa saja yang telah menghabiskan waktu di sini dan tahu bagaimana hal-hal umumnya bekerja. Bahkan jika Anda merasa tidak terlalu mengenal dengan baik, Anda bisa mengikuti, membaca, dan belajar, kemudian mulai membantu di kemudian hari!
+osu! memiliki tim peninjau yang berdedikasi untuk memeriksa apakah beatmap-beatmap siap untuk dipindahkan ke bagian peringkat dan dimainkan oleh semua orang. Kamu juga dapat itu berkontribusi di sektor ini:
```
Comment (wowcake) [inline wiki/How_you_can_help!/id.md:39]:
```suggestion
osu! memiliki tim peninjau yang berdedikasi untuk memeriksa apakah beatmap-beatmap siap untuk dipindahkan ke bagian peringkat dan dimainkan oleh semua orang. Kamu juga dapat ikut berkontribusi di sektor ini:
```

File: wiki/How_you_can_help!/id.md [hunk 10]
```diff
@@ -1,84 +1,67 @@
...
 
-### Dimulai dari mana
+- Lihat atau cari daftar [Pending Beatmaps](https://osu.ppy.sh/beatmapsets?s=pending) untuk artis atau lagu yang Kamu suka dan/atau ingin Kamu kontribusikan.
+- Mainkan beatmap [Pending atau Work-in-progress](/wiki/Beatmap/Category#work-in-progress-and-pending) dan tinggalkan umpan balik di utas diskusi beatmap.
```
Comment (wowcake) [inline wiki/How_you_can_help!/id.md:42]:
```suggestion
- Mainkan beatmap [Pending atau Work-in-progress](/wiki/Beatmap/Category#work-in-progress-and-pending) dan tinggalkan umpan balik di thread diskusi beatmap.
```

File: wiki/How_you_can_help!/id.md [hunk 11]
```diff
@@ -1,84 +1,67 @@
...
+- Baca [Panduan Modding](/wiki/Modding) dan [Ranking Kriteria](/wiki/Ranking_Criteria) untuk mengetahui dasar-dasar teknis modding dan apa yang membuat beatmap rankable.
+- Bergabunglah dengan kanal obrolan `#mapping` dan` #modreqs` dalam gim dan ikuti percakapan.
+- jika Kamu punya waktu dan dedikasi untuk gim, buat sebuah utas [Modding Queue](https://osu.ppy.sh/community/forums/60) untuk memberikan tempat kepada pembuat beatmap meminta mod dari Kamu.
```
Comment (wowcake) [inline wiki/How_you_can_help!/id.md:45]:
```suggestion
- Jika Kamu punya waktu dan dedikasi untuk gim, cobalah membuat sebuah [Modding Queue](https://osu.ppy.sh/community/forums/60) thread untuk memberikan tempat bagi para pembuat beatmap meminta mod dari Kamu.
```

File: wiki/How_you_can_help!/id.md [hunk 12]
```diff
@@ -1,84 +1,67 @@
...
+## Membantu atau bergabung dengan proyek komunitas yang dijalankan pengguna
 
-## Bantu periksa/mod beatmap!
+osu! memiliki komunitas teknis dan kompetitif yang berkembang pesat. Mulai dari penggemar, hobi, hingga veteran berpengalaman rhythm gim. Ada banyak proposal dan proyek yang dibuat khusus untuk osu! dan Kamu dapat membantu mereka, atau mendorong masa depan osu! dengan proposal/proyek Kamu sendiri!
 
-Meskipun kami telah mengatur tim yang menangani proses persetujuan beatmap, tidak ada yang bisa menghentikan siapa pun untuk membantu. Anda dapat mulai semudah memposting di kiriman beatmap pending, meninggalkan umpan balik positif atau kritik konstruktif Anda. Ketika Anda menjadi lebih baik, tidak ada salahnya untuk Anda bergabung menjadi salah satu tim persetujuan beatmap :).
+Di bawah ini beberapa cara Kamu dapat berkontribusi:
 
-### Dimulai dari mana
+- Jelajahi forum [Turnaments](https://osu.ppy.sh/community/forums/55). Turnamen membutuhkan banyak waktu dan pekerjaan untuk dapat diatur tanpa hambatan, dan Kamu dapat membantu dengan menjadi juri penonton dalam beberapa kasus.
```
Comment (wowcake) [inline wiki/How_you_can_help!/id.md:53]:
```suggestion
- Jelajahi forum [Turnamen](https://osu.ppy.sh/community/forums/55). Turnamen membutuhkan banyak waktu dan pekerjaan untuk dapat diatur tanpa hambatan, dan Kamu dapat membantu dengan menjadi juri penonton dalam beberapa kasus.
```

File: wiki/How_you_can_help!/id.md [hunk 13]
```diff
@@ -1,84 +1,67 @@
...
-
-### Dimulai dari mana
+Dimulai pada tahun 2017, proyek osu!lazer saat ini menjadi fokus osu! team, bersama dengan situs web baru yang Kamu lihat saat ini! Jika Kamu ingin membantu menjadikan proyek ini lebih baik dan membawa kita lebih dekat ke masa depan osu!, ada beberapa hal yang dapat Kamu lakukan, tetapi di bawah ini hanya beberapa contoh:
 
 - Bersikap ramah satu sama lain. Hidup terlalu pendek untuk diperdebatkan.
-- Jangan mengeluhkan pemain yang melakukan kecurangan. [Laporkan saja dan lanjutkan hari Anda](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play). Mereka mungkin telah mengambil peringkat teratas Anda, tetapi sesederhana `!report <username> sepertinya melakukan kecurangan` dapat memperbaikinya dalam hitungan menit hingga berjam-jam.
-- Gunakan cara Anda untuk menengahi situasi negatif dan ubah menjadi hal yang lebih positif.
+- Jangan mengeluhkan jika terdapat pemain yang melakukan kecurangan. [Laporkan saja dan lanjutkan harimu](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play), sederhananya menggunakan perintah `!report <username> sepertinya melakukan kecurangan` dapat memperbaikinya masalah Kamu dalam beberapa menit hingga jam.
```
Comment (wowcake) [inline wiki/How_you_can_help!/id.md:64]:
```suggestion
- Jangan mengeluh tentang pemain yang melakukan kecurangan. [Laporkan saja mereka dan lanjutkan harimu](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play), sederhananya menggunakan perintah `!report <username> sepertinya melakukan kecurangan` dapat memperbaiki masalah Kamu dalam beberapa menit hingga jam.
```

**Review by wowcake — CHANGES_REQUESTED** [review]

File: wiki/How_you_can_help!/id.md (hunk 12 above)
Comment (mousesalt) [reply wiki/How_you_can_help!/id.md:53]: i will keep this same to main web page

File: wiki/How_you_can_help!/id.md [hunk 14]
```diff
@@ -1,84 +1,67 @@
...
+osu! memiliki tim peninjau yang berdedikasi untuk memeriksa apakah beatmap-beatmap siap untuk dipindahkan ke bagian peringkat dan dimainkan oleh semua orang. Kamu juga dapat ikut berkontribusi di sektor ini:
 
-### Dimulai dari mana
+- Lihat atau cari daftar [Pending Beatmaps](https://osu.ppy.sh/beatmapsets?s=pending) untuk artis atau lagu yang Kamu suka dan/atau ingin Kamu kontribusikan.
+- Mainkan beatmap [Pending atau Work-in-progress](/wiki/Beatmap/Category#work-in-progress-and-pending) dan tinggalkan umpan balik di utas diskusi beatmap.
+- Baca [Panduan Modding](/wiki/Modding) dan [Ranking Kriteria](/wiki/Ranking_Criteria) untuk mengetahui dasar-dasar teknis modding dan apa yang membuat beatmap rankable.
+- Bergabung dan ikutilah percakapan pada kanal obrolan `#mapping` dan` #modreqs` dalam osu!.
+- Jika Kamu punya waktu dan dedikasi untuk osu!, cobalah membuat sebuah utas [Modding Queue](https://osu.ppy.sh/community/forums/60) untuk memberikan tempat bagi para pembuat beatmap meminta mod dari Kamu.
```
Comment (ZeroPyrozen) [inline wiki/How_you_can_help!/id.md:45]:
```suggestion
- Jika Anda punya waktu dan dedikasi untuk osu!, cobalah membuat sebuah utas [Modding Queue](https://osu.ppy.sh/community/forums/60) untuk memberikan tempat bagi para pembuat beatmap meminta mod dari Anda.
```

File: wiki/How_you_can_help!/id.md [hunk 15]
```diff
@@ -1,84 +1,67 @@
...
 
-- Pilih sebuah forum: [Technical Support](https://osu.ppy.sh/community/forums/5), [Beatmapping Help](https://osu.ppy.sh/community/forums/56), [Storyboarding](https://osu.ppy.sh/community/forums/20) atau [Gameplay and Rankings](https://osu.ppy.sh/community/forums/13) berdasarkan apa yang Anda anggap paling menarik.
-- Tandai forum sebagai telah dibaca menggunakan tautan di kanan atas.
-- Ikuti setiap ada kiriman baru, baca satu per satu dan bantu jika Anda bisa. Anda dapat menggunakan ikon yang belum dibaca untuk memastikan Anda mengikuti!
-- Baca [panduan yang tersedia](/wiki/Guides) dan pertajam pengetahuan osu! Anda. Kalau bisa, berkontribusi pada panduan tersebut!
+## Membantu atau bergabung dengan proyek komunitas yang dijalankan pengguna
 
-## Bantu periksa/mod beatmap!
+osu! memiliki komunitas teknis dan kompetitif yang berkembang pesat. Mulai dari penggemar, hobi, hingga veteran berpengalaman rhythm gim. Ada banyak proposal dan proyek yang dibuat khusus untuk osu! dan Kamu dapat membantu mereka, atau mendorong masa depan osu! dengan proposal/proyek Kamu sendiri!
```
Comment (ZeroPyrozen) [inline wiki/How_you_can_help!/id.md:49]:
```suggestion
osu! memiliki komunitas teknis dan kompetitif yang berkembang pesat. Mulai dari penggemar, hobi, hingga veteran berpengalaman gim ritme. Ada banyak proposal dan proyek yang dibuat khusus untuk osu! dan Anda dapat membantu mereka, atau mendorong masa depan osu! dengan proposal/proyek Anda sendiri!
```

File: wiki/How_you_can_help!/id.md [hunk 16]
```diff
@@ -1,84 +1,67 @@
...
 
-Meskipun kami telah mengatur tim yang menangani proses persetujuan beatmap, tidak ada yang bisa menghentikan siapa pun untuk membantu. Anda dapat mulai semudah memposting di kiriman beatmap pending, meninggalkan umpan balik positif atau kritik konstruktif Anda. Ketika Anda menjadi lebih baik, tidak ada salahnya untuk Anda bergabung menjadi salah satu tim persetujuan beatmap :).
+Di bawah ini beberapa cara Kamu dapat berkontribusi:
 
-### Dimulai dari mana
+- Jelajahi forum [Turnaments](https://osu.ppy.sh/community/forums/55). Turnamen membutuhkan banyak waktu dan pekerjaan untuk dapat diatur tanpa hambatan, dan Kamu dapat membantu dengan menjadi juri penonton dalam beberapa kasus.
```
Comment (ZeroPyrozen) [inline wiki/How_you_can_help!/id.md:53]:
```suggestion
- Jelajahi forum [Turnaments](https://osu.ppy.sh/community/forums/55). Turnamen membutuhkan banyak waktu dan pekerjaan untuk dapat diatur tanpa hambatan, dan Anda dapat membantu dengan menjadi juri penonton dalam beberapa kasus.
```

**Review by ZeroPyrozen — CHANGES_REQUESTED** [review]

**Review by ZeroPyrozen — APPROVED** [review]
All good 👍

**Review by TicClick — APPROVED** [review]

**Pull request merged**